{"name": "Test of creating an order with incorrect ingredients", "status": "passed", "start": 1713953695077, "stop": 1713953695211, "uuid": "c9726421-0fbc-4c0b-ad9c-9976b7079193", "historyId": "fae4207df7a502a11a4347d0c9b22840", "testCaseId": "fae4207df7a502a11a4347d0c9b22840", "fullName": "tests.test_create_order.TestApiCreateOrder#test_create_order_bad_ingredients_error", "labels": [{"name": "feature", "value": "Проверяем создание заказа"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestApiCreateOrder"}, {"name": "host", "value": "IZY-PC"}, {"name": "thread", "value": "25448-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}